kaiju/SectionHeader.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "SectionHeader",
+  "library": "terra-section-header",
+  "display": "Section Header",
+  "description": "Presentational header for sections",
```

Comment:
Accepting description donations

Reply:
A header for distinguishing a section from other sections.

![](https://media.giphy.com/media/12ETcAyWE6TjHy/giphy.gif)